specs/altair/sync-protocol.md

```diff
@@ -237,13 +269,12 @@ def validate_light_client_update(store: LightClientStore,
 
 ```python
 def apply_light_client_update(store: LightClientStore, update: LightClientUpdate) -> None:
```

Comment:
[not strongly] I feel this function could be renamed to something like describe what it actually does. e.g., `update_store_finality` or so.

Reply:
Hmm, agree it's a bit vague, but it updates all of sync committee, finalized header and to some extent optimistic header, and only mentioning `finality` sort of hides the other side effects.

Reply:
that's also true. 😅